templates/main/view_test.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
     <head>
-        <meta charset="utf-8" />
         <title>View test</title>
-        <link rel="stylesheet" href="css/main.css">
         <script src="js/view_test.js" defer></script>
+        <?php include "head-inc.php" ?>
     </head>
     <body>
+        <?php include "navbar-inc.php" ?>
         <div id="container">
             <h1><?=$params["title"]?></h1>
             <?php if ($params["is_viewing_results"]): ?>
```

Comment:
I think we can refactor the code so that we would not need to do all that copy-pasting of `<?php include "head-inc.php" ?>` and `<?php include "navbar-inc.php" ?>`.
What we need to do is:
- Use the following html template, located in `templates/main` and named for example `main.html`:
``` html
<!DOCTYPE html>
<html lang="en">
    <head>
        <title><?=$this->title?></title>
        <link rel="stylesheet" href="css/main.css">
        <script src="js/<?=$this->module?>.js" defer></script>
        <?php include "head-inc.php" ?>
    </head>
    <body>
        <?php include "navbar-inc.php" ?>
        <?php include View::TEMPLATE_DIR . "/modules/" . $this->module . ".html" ?>
    </body>
</html>
```
- Move all current .html files into a subfolder in `templates/main` called `modules`
- Make sure all modules have their corresponding `.js` files in the `js` folder
- Refactor `View.class.php` so that we now have `$title` as well as a `$module` property and also replace the `include` statement in `View::send` with
```php
include self::TEMPLATE_DIR . "/main.html";
```
or
```php
include self::TEMPLATE_DIR . "/". SETTINGS["TEMPLATE_NAME"] . ".html";
```